### ISSUE TITLE
Hide buttons while sending file 

### ISSUE THREAD

**Issue by ddci** [ddci/vkfilebot#1]
Need to hide buttons in first message after choosing _"As file"_

**Comment by ddci** [comment]
Closed with commit b245657d43031ffa3e6424b32d94b6a8c5ce059e
```
text += "Размер фаила:" + "<b>" + " " "%.3f" % size + "MB" + "</b>" + "\n"
        bot.send_message(cid, usersDownLink[lastChoosedFile[cid]], parse_mode="HTML", reply_markup=hideBoard)
        userStep[cid] = 0
```

**Comment by ddci** [comment]
Closed with b245657d43031ffa3e6424b32d94b6a8c5ce059e
```
text += "Размер фаила:" + "<b>" + " " "%.3f" % size + "MB" + "</b>" + "\n"
bot.send_message(cid, usersDownLink[lastChoosedFile[cid]], parse_mode="HTML", reply_markup=hideBoard)
        userStep[cid] = 0
```
